prime-router/src/test/kotlin/cli/ValidateSettingCommandsTest.kt

```diff
@@ -0,0 +1,134 @@
+        val validateCmd = ValidateSettingCommands()
+        validateCmd.parse(
+            listOf(
+                "-i", "./src/test/unit_test_files/invalid_setting_jurisdiction_vs_state_code_county_name.yml",
+            "-s", "settings/schemas/settings.json"
+            )
+        )
+        val result = validateCmd.validate()
+
+        assertTrue(result.size == 8)
+        assertTrue(
+            result.elementAt(0).toString()
+                .contains("[0]: For STATE jurisdiction, countyName must NOT present")
+        )
+        assertTrue(
+            result.elementAt(1).toString()
+            .contains("[1]: For STATE jurisdiction, stateCode must present")
+        )
+        assertTrue(
+            result.elementAt(2).toString()
+                .contains("[2]: For FEDERAL jurisdiction, stateCode must NOT present")
+        )
+        assertTrue(
+            result.elementAt(3).toString()
+                .contains("[3]: For FEDERAL jurisdiction, countyName must NOT present")
+        )
+        assertTrue(
+            result.elementAt(4).toString()
+            .contains("[4]: For COUNTY jurisdiction, both stateCode and countyName is required")
+        )
+        assertTrue(
+            result.elementAt(5).toString()
+            .contains("[5]: For COUNTY jurisdiction, both stateCode and countyName is required")
+        )
+        assertTrue(
+            result.elementAt(6).toString()
+            .contains("[5]: For COUNTY jurisdiction, both stateCode and countyName is required")
+        )
+        assertTrue(
+            result.elementAt(7).toString()
+            .contains("[6]: For COUNTY jurisdiction, both stateCode and countyName is required")
+        )
+    }
+
+    @Test
+    fun `test validate single sender with sub schema sender json`() {
+        val validateCmd = ValidateSettingCommands()
+        validateCmd.parse(
+            listOf(
+                "-i", "./src/test/unit_test_files/one_sender_waters.yml",
+                "-s", "settings/schemas/sender.json"
+            )
+        )
+        val result = validateCmd.validate()
+
+        assertTrue(result.size == 1)
+        assertTrue(result.elementAt(0).toString().contains("name: null found, string expected"))
+    }
+
+    @Test
+    fun `test validate single receiver with sub schema receiver json`() {
+        val validateCmd = ValidateSettingCommands()
+        validateCmd.parse(
+            listOf(
+                "-i", "./src/test/unit_test_files/one_receiver_waters.yml",
+                "-s", "settings/schemas/receiver.json"
+            )
+        )
+        val result = validateCmd.validate()
+
+        assertTrue(result.size == 1)
+        assertTrue(
+            result.elementAt(0).toString()
+        .contains("conditionFilter not allowed for topic: 'covid-19', 'monkeypox', 'test'")
+        )
+    }
+
+    @Test
+    fun `test validate single organization with sub schema organization json`() {
```

Comment:
Thank you for writing so many tests!

Reply:
> Thank you for writing so many tests!

yep, previously the putSetting is not covered, so now got it covered